docs/install/assets/eventing.md

```diff
@@ -0,0 +1,11 @@
+# Knative Serving installation files (0.21 releases)
+
+| File name | Description | Dependencies|
+| --- | --- | --- |
+| eventing-core.yaml | Knative Serving core components. |  none |
```

Comment:
```suggestion
| eventing-core.yaml | Knative Eventing core components. |  eventing-crds.yaml |
```

Reply:
Whoops, thanks for catching the `Serving` -> `Eventing`. That was my bad.
Regarding the dependency, does the comment above make sense ? https://github.com/knative/docs/pull/3304#discussion_r586638503

Reply:
Carlos explained what I was thinking.